news/2023/2023-11-08-OWC2023-RO16-recap.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "osu! World Cup 2023: Round of 16 Recap"
+date: 2023-11-08 16:30:00 +0000
+---
+
+Want to stay up to date with all the latest osu! World Cup happenings, or maybe just look back at the action packed weekend? [Junihuhn](https://osu.ppy.sh/users/4182339) is here to deliver!
+
+![](/wiki/Tournaments/OWC/2023/img/owc2023-banner.jpg)
+
+As previously mentioned in [this news post](https://osu.ppy.sh/home/news/2023-08-21-contributing-to-the-osu-news-feed), this year you can expect a weekly rundown of the osu! World Cup stages, starting with the Round of 16. Hosted by the one and only Junihuhn, they will be taking the best moments of the previous week's stage and *burning them into the retina's of your eyes*.
+
+Also, as is tradition by now they will also be doing a video series on their [YouTube channel](https://www.youtube.com/@Junihuhn/) (which is a mandatory viewing for all osu! World Cup enthusiasts). You can find the [Round of 32 recap](https://www.youtube.com/watch?v=ydOlxHQg0UM) from the previous week on there, or you can check out the Round of 16 recap in a video format below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/SlWKKA-ltZY" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Without further ado, let's look at some highlighted matches from the Round of 16!
+
+### ::{ flag=DE }:: Germany vs. ::{ flag=CA }:: Canada
+
+Yes, you are reading this correctly.
+
+Due to Canada's unusual 11th seed placing ([due to some players playing qualifiers at TwitchCon with technical issues](https://twitter.com/xootynator/status/1716179172270166224)), this battle of tournament veterans happened in RO16. The spectators were expecting a coin flip match, well represented by the [osu! World Cup pick'ems](https://pickem.hwc.hr/tournaments/130)[^1]. The North American squad had the better start going up 3-1 including an important break point on [Mysteka - Jerrybeans](https://osu.ppy.sh/beatmapsets/1860553#osu/3825151). Germany responded quickly on the low approach rate map [jun with Alison - SUNKiSS DROP ~jun side~](https://osu.ppy.sh/beatmapsets/1498309#osu/3070802), claiming the break point right back. [Criller](https://osu.ppy.sh/users/8116659)'s double S and [Shiox](https://osu.ppy.sh/users/11921197)'s 1x100 score provided much needed confidence.
+
+However, the slug fest of break points continued on a highly contested [Yamamoto Mineko - Presea](https://osu.ppy.sh/beatmapsets/2082024#osu/4360325) pick. Eventually, this turned out to be the crucial point for Canada, and they converted their first match point into the seed upset 5-2 victory against the bronze medallist of 2022.
+
+[^1]: Scroll to the bottom of the page, select `Round of 16` and find the match marked `X`.
```

Comment:
Notes are usually found at the bottom of the post with a level 2 header, i.e. `## Notes` (for wiki articles at least).

That said, I'm not 100% sure that's the reason for it not working as footnotes in news posts are very rare

Reply:
the only reason for the footnote is that the website does not make it clear where you can see previous rounds, I literally couldn't find it for 5 mins lol

Reply:
![image](https://github.com/ppy/osu-wiki/assets/24534022/de9597d5-10a9-4990-9376-4422380967c9)

I'd like to hear opinions on whether we can leave the link as is without the footnote (due to my concern voiced above) or if we need to figure out something else

Reply:
addressed in https://github.com/ppy/osu-wiki/pull/10408/commits/e252dcac10f70e6bc12be6d55eb70efd83edc70f, should be sufficient without breaking the overall formatting of the post